pkg/starboard/config_test.go

```diff
@@ -324,67 +325,204 @@ func TestConfigManager_Read(t *testing.T) {
 
 func TestConfigManager_EnsureDefault(t *testing.T) {
 
-	t.Run("Should create ConfigMap with default values, and empty secret", func(t *testing.T) {
+	t.Run("Should create ConfigMaps and Secret", func(t *testing.T) {
+		g := gomega.NewGomegaWithT(t)
```

Comment:
Gomega is more flexible and expressive than testify. We also use Gomega in integration tests so I'd like to stick to Gomega for new code.